spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationJob.scala

```diff
@@ -15,527 +15,38 @@
 
 package za.co.absa.enceladus.standardization
 
-import java.io.{PrintWriter, StringWriter}
-import java.text.MessageFormat
-import java.time.Instant
-import java.util.UUID
-
-import com.typesafe.config.ConfigFactory
-import org.apache.spark.SPARK_VERSION
-import org.apache.spark.sql.types.{StructField, StructType}
-import org.apache.spark.sql.{Column, DataFrame, DataFrameReader, SparkSession}
-import org.slf4j.LoggerFactory
-import za.co.absa.atum.AtumImplicits
-import za.co.absa.atum.core.Atum
-import za.co.absa.enceladus.common.RecordIdGeneration.{IdType, _}
-import za.co.absa.enceladus.common._
-import za.co.absa.enceladus.common.plugin.PostProcessingService
-import za.co.absa.enceladus.common.plugin.menas.{MenasPlugin, MenasRunUrl}
-import za.co.absa.enceladus.common.version.SparkVersionGuard
+import org.apache.spark.sql.SparkSession
 import za.co.absa.enceladus.dao.MenasDAO
-import za.co.absa.enceladus.dao.auth.MenasCredentials
-import za.co.absa.enceladus.dao.rest.{MenasConnectionStringParser, RestDaoFactory}
-import za.co.absa.enceladus.model.Dataset
-import za.co.absa.enceladus.standardization.interpreter.StandardizationInterpreter
-import za.co.absa.enceladus.standardization.interpreter.stages.PlainSchemaGenerator
-import za.co.absa.enceladus.utils.config.{ConfigReader, SecureConfig}
+import za.co.absa.enceladus.dao.rest.RestDaoFactory
+import za.co.absa.enceladus.standardization.config.StandardizationConfig
 import za.co.absa.enceladus.utils.fs.FileSystemVersionUtils
-import za.co.absa.enceladus.utils.general.ProjectMetadataTools
-import za.co.absa.enceladus.utils.performance.{PerformanceMeasurer, PerformanceMetricTools}
-import za.co.absa.enceladus.utils.schema.{MetadataKeys, SchemaUtils, SparkUtils}
-import za.co.absa.enceladus.utils.time.TimeZoneNormalizer
+import za.co.absa.enceladus.utils.modules.SourcePhase
 import za.co.absa.enceladus.utils.udf.UDFLibrary
-import za.co.absa.enceladus.utils.unicode.ParameterConversion._
-import za.co.absa.enceladus.utils.validation.ValidationException
-
-import scala.collection.immutable.HashMap
-import scala.util.control.NonFatal
-import scala.util.{Failure, Success, Try}
 
-object StandardizationJob {
-  TimeZoneNormalizer.normalizeJVMTimeZone()
-
-  private val log = LoggerFactory.getLogger(this.getClass)
-  private val conf = ConfigFactory.load()
-  private val confReader: ConfigReader = new ConfigReader(conf)
-  private val menasBaseUrls = MenasConnectionStringParser.parse(conf.getString("menas.rest.uri"))
-  private final val SparkCSVReaderMaxColumnsDefault: Int = 20480
+object StandardizationJob extends StandardizationExecution {
```

Comment:
I am trying to run PR, but there is a problem with running standardization:
```
20/07/13 13:17:54 INFO core.SparkLineageInitializer$: Spline successfully initialized. Spark Lineage tracking is ENABLED.
Exception in thread "main" java.lang.IllegalStateException: Control framework tracking is not initialized.
	at za.co.absa.atum.core.Atum$.preventNotInitialized(Atum.scala:228)
	at za.co.absa.atum.core.Atum$.setAllowUnpersistOldDatasets(Atum.scala:122)
	at za.co.absa.enceladus.common.CommonJobExecution$class.prepareJob(CommonJobExecution.scala:100)
	at za.co.absa.enceladus.standardization.StandardizationJob$.prepareJob(StandardizationJob.scala:26)
	at za.co.absa.enceladus.standardization.StandardizationJob$.main(StandardizationJob.scala:38)
	at za.co.absa.enceladus.standardization.StandardizationJob.main(StandardizationJob.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52)
	at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:845)
	at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:161)
	at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:184)
	at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:86)
	at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:920)
	at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:929)
	at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
```

Reply:
True, found the missing line